test/cpp/api/special.cpp

```diff
@@ -13,3 +13,35 @@ TEST(SpecialTest, special) {
     auto t = torch::randn(128, torch::kDouble);
     torch::special::gammaln(t);
 }
+
+template <typename Op, typename OpOut>
```

Comment:
What's up with this new test? Is this maybe more appropriate for a separate PR?

Reply:
Removed.